README.md

```diff
@@ -75,7 +75,9 @@ then [check out the "help wanted" label](https://github.com/palantir/blueprint/l
 Builds are orchestrated via [Nx's task runner](https://nx.dev/getting-started/intro) and NPM scripts.
 [Lerna-Lite](https://github.com/lerna-lite/lerna-lite) is used to prepare releases.
 
-**Prerequisites**: Node.js v18+ (see version specified in `.nvmrc`), Yarn v4.x (see version specified in `package.json`)
+**Prerequisites**: Node.js v20.11+ (see version specified in `.nvmrc`), Yarn v4.x (see version specified in `package.json`)
+- note `yarn` may need to be updated from outside of the Blueprint repo or you may see an error like `This project's package.json defines "packageManager": "yarn@4.0.2". However the current global version of Yarn is 1.22.22`
+- update `yarn` with `yarn set version <version from package.json>`
```

Comment:
The `packageManager` field is for `corepack`. Since it's set up in the Blueprint repo, should we encourage developers to run `corepack enable` instead of `yarn set version`?

Reply:
hmm we do have that below but I don't think that worked for me. I needed to back out of BP and run this before I was able to go back into BP and run `yarn` to install deps

no urgency to merge this and would err towards not doing anything, I may reclone and see what happens but wonder if now that I have yarn 4 downloaded it would work